backend/src/mastodon/sql/instance.ts

```diff
@@ -0,0 +1,10 @@
+// Prepared statements for Mastodon Instance API endpoints
+export const instanceStatisticsQuery = (origin: string): string => {
```

Comment:
```suggestion
export const instanceStatisticsQuery = (domain: string): string => {
```

For consistency, we called it `domain`. Could you please rename them?

Reply:
Absolutely. I also added inline documentation to clarify that the parameter expected is an **origin** (i.e. must include the `https://`)